settings.py

```diff
@@ -37,7 +37,9 @@
 # Prevent writing clear email for avoiding robots on Github sources.
 EDITORS_EMAIL_DEFAULT = '@'.join(['editors', 'cafebabel.com'])
 EDITOR_EMAILS = {
-    'en': '@'.join(['en', 'cafebabel.co.uk']),
-    'fr': '@'.join(['fr', 'cafebabel.fr']),
-    'it': '@'.join(['it', 'cafebabel.it']),
+    'fr': '@'.join(['redaction', 'cafebabel.com']),
```

Comment:
Not related to that PR but rethinking about these emails-related stuff: I think we should configure with a catch-all email for now at least to get feedback on the current website without spamming anyone.